t/plugin/limit-count-redis.t

```diff
@@ -209,3 +212,69 @@ passed
 [404, 503, 404, 503, 503]
 --- no_error_log
 [error]
+
+
+
+=== TEST 6: set route, with redis host, port and wrong password
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                [[{
+                    "plugins": {
+                        "limit-count": {
+                            "count": 2,
+                            "time_window": 60,
+                            "rejected_code": 503,
+                            "key": "remote_addr",
+                            "policy": "redis",
+                            "redis_host": "127.0.0.1",
+                            "redis_port": 6379,
+                            "redis_timeout": 1001,
+                            "redis_password": "WRONG_foobared"
+                        }
+                    },
+                    "upstream": {
+                        "nodes": {
+                            "127.0.0.1:1980": 1
+                        },
+                        "type": "roundrobin"
+                    },
+                    "uri": "/hello_new"
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.print(body)
+        }
+    }
+--- request
+GET /t
+--- error_code eval
+200
+--- no_error_log
+[error]
+
+
+
+=== TEST 7: request for TEST 6
+--- request
+GET /hello_new
+--- error_code eval
+500
+--- response_body
+{"1":"ERR invalid password","error_msg":"failed to limit count: "}
```

Comment:
Is the response_body wrong? should be `"error_msg":"failed to limit count:  ERR invalid password"`?

Reply:
> Is the response_body wrong? should be `"error_msg":"failed to limit count: ERR invalid password"`?

Not true. `Failed to limit req: ERR invalid password, client: 127.0.0.1 ` etc was found in error log. `{"1":"ERR invalid password","error_msg":"failed to limit count: "}` is the correct response

Reply:
the response is confusing, where it comes from?

Reply:
`"1":"ERR invalid password"`

this is unacceptable, we need to fix it.